tideways/config.php-profiler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+$config = array(
+    'save.handler' => \Xhgui\Profiler\Profiler::SAVER_PDO,
```

Comment:
refrain yourself using direct writes to xhgui database. the recommended saver is upload saver.

writing directly to xhgui data store will be removed in the future.

do read the project readme.

Reply:
So upload way it is just calling XHgui to save data and not interacting with sqlite.
For our needs this can slowdown a bit as it is a vagrant machine with all the tool inside, change it is not a problem but for our case maybe adds a layer of complexity?

Reply:
You can use FILE saver then, so there's less overhead, just need to figure how to somehow to manually invoke import. If they share the filesystem perhaps can use cron to import things periodically (and rename the input file to prevent double imports)?

writing directly to xhgui database prevents making internal changes to xhgui, so therefore such use is deprecated (but it's unknown yet when the support be dropped)

Reply:
How much slower is using the upload URL? Could it be made faster? It shouldn't be an issue with an asynchronous non-blocking request, especially since it's a local loopback so it's just the sending and initial connection cost

Reply:
I've created an issue for this so that this doesn't block the merging of this PR at: https://github.com/Varying-Vagrant-Vagrants/VVV/issues/2251 I'd really like to futureproof support so things don't break down the line